src/System.IO.Pipelines/ref/System.IO.Pipelines.csproj

```diff
@@ -16,4 +16,7 @@
   <ItemGroup>
     <ProjectReference Include="..\..\System.Buffers\ref\System.Buffers.csproj" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == 'netstandard' OR '$(TargetsNetFx)' == 'true'">
```

Comment:
Can we just include this source file https://github.com/dotnet/corefx/blob/master/src/Common/src/CoreLib/System/Threading/Tasks/Sources/ManualResetValueTaskSourceCore.cs

Reply:
Tried that, that made packaging even more unhappy as the type was public but not defined in the ref project 

@stephentoub would an internal version of `ManualResetValueTaskSourceCore` for netstd make sense?

Reply:
> would an internal version of ManualResetValueTaskSourceCore for netstd make sense?

I don't think we should do that; I'd like to minimize the number of copies we have floating around in this or that package.  If we really need to use it, I assume we could reference the async package that exposes it.

Reply:
>  I assume we could reference the async package that exposes it.

After 30 tries with packaging commits; I think I've done that successfully. There's something unusual with the `System.IO.Pipelines` packaging; `System.Text.Json` looks to use it happily and adding it to WebSockets was quite straightforward.